components/Completed/Completed.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Completed = () => {
+  return (
+    "hi"
+  );
```

Comment:
return something meaningful so you can verify that routing works.

Reply:
Done.